canonical/mir-server.io
# WD-3563 - Make the docs navigation expandable

## Done
Added the required JS and template macros to make the navigation expandable

## QA
1. Open the demo
2. Go to docs and see that the navigation is expandable
3. Use it and see it works as expected
4. Refresh the page to ensure the navigations renders in the correct state
5. Check different viewports

Fixes https://github.com/canonical/mir-server.io/issues/219
